Unaccountable-Magpie/BangazonAPI2
# Jr controller2

Purpose:
added OrdersCompleted to orders table and IsDeleted to others


How it fits in context of the project:

needed to columns

Specific Feature affected:
sql file


How to test (Be thorough!):

1.open BangazonApi.sql in sql manager

2.execute

3.make sure the tables have the new columns
